packages/request-manager/src/interceptor.ts

```diff
@@ -153,9 +154,16 @@ const overloadWebsocketHandshake = (
     options?: http.RequestOptions | ((r: http.IncomingMessage) => void),
     callback?: unknown,
 ) => {
+    if (typeof url !== 'object') return;
```

Comment:
Because of this condition, `typeof url === 'object'` in the if block below is now redundant.

Reply:
[ac1ef86](https://github.com/trezor/trezor-suite/pull/7986/commits/ac1ef86199d43ec8d25fdad8278f5ed79d3fcf1f)